### PR TITLE
ACMS-3680: Update ACMS Common to fix GitLab CI.

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**
Fixes #[ACMS-3680](https://acquia.atlassian.net/browse/ACMS-3680)

**Proposed changes**
Update ACMS Common to fix GitLab CI.

**Alternatives considered**
NA

**Testing steps**
Review CI.

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by deepakmishra2 — APPROVED** [review]

**Review by apathak18 — APPROVED** [review]

File: modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php [hunk 1]
```diff
@@ -102,21 +102,36 @@ public function testAcquiaConfigSyncOverriddenPage() {
     $assert_session = $this->assertSession();
     $assert_session->statusCodeEquals(200);
 
-    // Asset that overridden configuration appears on overridden tab.
-    $assert_session->elementExists('xpath', "//table/tbody/tr/td[contains(text(),'taxonomy.vocabulary.tags')]");
-    $assert_session->elementExists('xpath', "//table/tbody/tr/td/span[contains(text(),'87  %')]");
+    // Asset that overridden configuration table exist.
+    $assert_session->elementExists('css', 'table.responsive-enabled');
 
-    // Asset that unchanged configuration does not appear on overridden tab.
```
Comment (chandan-singh7929) [inline modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php:109]: Let's not remove this test.

Comment (rajeshreeputra) [reply modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php:109]: The taxonomy-related configuration is currently not being overridden by the ACMS Common module with testing profile. see screenshot
ACMS-overriden-config

File: modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php [hunk 2]
```diff
@@ -102,21 +102,36 @@ public function testAcquiaConfigSyncOverriddenPage() {
...
-    $assert_session->elementNotExists('xpath', "//table/tbody/tr/td[contains(text(),'taxonomy.vocabulary.categories')]");
```
Comment (chandan-singh7929) [inline modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php:110]: This one as well, we are making sure that unchanged configurations appears on unchanged tab here.

Comment (rajeshreeputra) [reply modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php:110]: Done.

File: modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php (hunk 1 above)
Comment (rajeshreeputra) [reply modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php:109]: Done.

File: modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php [hunk 3]
```diff
@@ -102,21 +102,36 @@ public function testAcquiaConfigSyncOverriddenPage() {
...
+
+    // Asset that overridden configuration table has body exist.
+    $assert_session->pageTextContains('editor.editor.filtered_html');
+    $assert_session->pageTextContains('editor.editor.full_html');
 
     $this->drupalGet('/admin/config/development/acquia-cms-support/unchanged-config');
     $assert_session->statusCodeEquals(200);
 
-    // Asset that unchanged configuration appears on unchanged tab.
-    $assert_session->elementExists('xpath', "//table/tbody/tr/td[contains(text(),'taxonomy.vocabulary.categories')]");
```
[the remark this answers is not in the document]
Comment (rajeshreeputra) [reply modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php:116]: Done.

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**